templates/security/certifications.html

```diff
@@ -35,33 +35,43 @@ <h2 class="p-heading--3">Deploy regulated and high security workloads</h2>
  <div class="row p-divider">
    <div class="col-4 p-divider__block">
      <h3>Comply with established security baselines</h3>
-     <p>The default configuration of Ubuntu balances between usability and security. However, systems carrying dedicated workloads can be further hardened to reduce their attack surface. We work with DISA to ensure STIG guides available for Ubuntu, and we provide OpenSCAP tooling and automation for the Industry accepted CIS benchmark. Available with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
+     <p>The default configuration of Ubuntu balances usability and security. However, systems carrying dedicated workloads can be further hardened to reduce their attack surface. Canonical works with DISA to ensure STIG guides are available for Ubuntu, and we provide OpenSCAP tooling and automation for the Industry accepted CIS benchmark. Available with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
      <a class="p-button--neutral" href="#compliance-profiles">See our compliance profiles</a>
    </div>
    <div class="col-4 p-divider__block">
-     <h3>Transparency</h3>
-     <p>Canonical is transparent on the Security Features we enable on Ubuntu. We have a <a href="/security/notices">public vulnerability disclosure policy</a> and known vulnerabilities are disclosed in <a href="/security/oval">Machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party tools.</p>
+     <h3>Know your defenses</h3>
+     <p>Each Ubuntu release enables state of the art protection against vulnerability exploitation and malware and we <a class="p-link--external" href="https://wiki.ubuntu.com/Security/Features">publicly detail our choices</a>. Canonical has a <a href="/security/notices">public vulnerability disclosure policy</a> and vulnerabilities are not only fixed with automated security updates and <a href="/security/livepatch">livepatches</a> but also publicly disclosed with our <a href="/security/notices">security notices</a>. We further provide <a href="/security/oval">machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party vulnerability management tools.</p>
```

Comment:
"public vulnerability disclosure policy" link is incorrect. In copy docs it's `/security/disclosure-policy`